docs/guides/rate-limits.mdx

```diff
@@ -1,74 +1,216 @@
+like brute-force and credential stuffing, while allowing for higher volumes of legitimate traffic from trusted sources. These
+types of attacks typically involve numerous attempts to guess credentials or exploit vulnerabilities, often originating from a
+limited set of IP addresses.
+
+#### Key Benefits
+
+1. **Enhanced Security:**
+
+   - Restricts the number of requests from specific sources within a given timeframe.
+   - Makes it significantly harder for attackers to succeed with brute-force or credential stuffing attacks.
+   - Strengthens the security of your Ory projects and protects sensitive user data.
+
+2. **Protection Against Malicious Bots:**
+
+   - Differentiates between genuine user traffic and potentially harmful bot activity.
+   - Analyzes request patterns to identify and block automated malicious activities.
+
+3. **Safeguarding Specific Endpoints:**
+
+   - Offers granular control over how each endpoint handles traffic and responds to potential threats.
+   - Allows fine-tuning of security measures for individual endpoints.
+   - Optimizes protection without compromising the user experience.
+
+4. **Fair Usage:**
+
+   - Complements Project Rate Limits in ensuring fair resource allocation.
+   - Contributes to a fairer and more stable platform for all users by mitigating abusive traffic.
+
+5. **Flexibility for High-Volume Legitimate Traffic:**
+   - Provides options for Enterprise and Growth customers to whitelist internal IPs for higher rate limits.
+   - Balances security needs with the requirements of high-volume legitimate traffic.
+
+### Important Notes on Rate Limit Rules
+
+#### Rule Management
+
+The Endpoint-Based Rate Limit rules are set and managed by Ory. These rules are not directly configurable by Enterprise and Growth
+customers yet.
+
+#### IP Whitelisting for Enterprise and Growth Customers
+
+Enterprise and Growth Customers have the option to whitelist internal IPs. This feature is designed for cases where these IPs
+generate high volumes of legitimate calls that might otherwise trigger rate-limit rules.
 
-For all projects in a workspace, the rate limit is the same as for the production environment. For example, on the Production
-plan, the rate limit would be 80 rps on the `/sessions/whoami` path for all projects combined on the workspace.
+- Whitelisted IPs are subject to the same rule structure but with higher limits.
+- To whitelist IPs, please create a support ticket with Ory.
+- This feature ensures that necessary high-volume traffic from trusted sources is not interrupted while maintaining strong
+  security measures.
 
-## Legacy plans
+### List of Rate Limit Rules
 
-The legacy Essential and Scale plans have the same rate limits as Production and Growth respectively.
+This section will be populated later with detailed documentation of the rules set by Ory, including information on standard limits
```

Comment:
Dunno how detailed we want/can have this section. We could mostly copy this table here https://www.notion.so/Rate-Limits-a0821f7f807f431880e571cf6eb8fb63#c27c0c7124334b4b89326b3e993f25f4

Reply:
We do this as a follow up pr to get this version out asap.